### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by hanxlin**
Helm-chart should only contain generic values. All CZI specific values should have been removed, but would appreciate another set of eyes to keep ones I might have missed.

### CODE REVIEW

**Comment by kuannie1** [comment]
I personally don't see any sensitive info. Were you able to test this helm chart?

File: charts/aws-config-generation/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: http
+              scheme: HTTP
+            failureThreshold: 3
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
```
Comment (jgadling) [inline charts/aws-config-generation/templates/deployment.yaml:83]: Can we make the liveness probe optional, and make the thresholds for both probes configurable?

File: charts/aws-config-generation/templates/network_policy.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-{{ $fullName }}-external
+  namespace: {{ .Values.namespace }}
+spec:
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 0.0.0.0/0
```
Comment (jgadling) [inline charts/aws-config-generation/templates/network_policy.yaml:13]: This should be configurable

File: charts/aws-config-generation/values.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+project: si
+owner: infra-eng@chanzuckerberg.com
+env: playground
```
Comment (jgadling) [inline charts/aws-config-generation/values.yaml:3]: We don't want these to be defaults for the chart

File: charts/aws-config-generation/templates/deployment.yaml (hunk 1 above)
Comment (hanxlin) [reply charts/aws-config-generation/templates/deployment.yaml:83]: we want liveness probe to be optional?

Comment (hanxlin) [reply charts/aws-config-generation/templates/deployment.yaml:83]: ok I guess it makes sense for a generic template

Comment (jgadling) [reply charts/aws-config-generation/templates/deployment.yaml:83]: Keep in mind a failing liveness probe will *restart the container*

There are many reasons an application can become unresponsive - poor thread/lock management, resource starvation, and failures in *upstream services.* Liveness probes are great for restarting an application that can't resolve an internal deadlock, but this behavior can cause cascading failures when a cluster is resource constrained. Some teams choose to use liveness probes sparingly, and usually with higher thresholds than readiness probes.

File: charts/aws-config-generation/values.yaml (hunk 3 above)
Comment (hanxlin) [reply charts/aws-config-generation/values.yaml:3]: included by accident, removed

**Pull request merged**